### PR TITLE
Addition of rules for the handling of bullying, abuse, and judicial proceedings

### PR DESCRIPTION
**Pull request by metal-crow**
Check one:
- [X] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s):
Improve handling of abuse and bullying claims, how the Judicial occurs for them, and the outcome of the judicial. This is done to prevent the silencing of abuse, and sidelining of bullying issues. Also serves to ensure impartiality and prevent the dropping of cases due to abusers in positions of power.

Change handling of cases where a Judicial occurs involving an e-board member. Needed because it is unlikely e-board can remain impartial when judging another e-board member.

### CODE REVIEW

**Comment by stevenmirabito** [comment]
My immediate reaction is that we do not want to touch this. Anything this would apply to should be immediately taken to ResLife.

**Comment by sgreene570** [comment]
I am on the same page as @stevenmirabito . Hard pass.  We should tread carefully.

**Comment by metal-crow** [comment]
I see. I agree, but that should be codified then, because as is that is not guaranteed to happen. I can rework this to specify that

**Comment by jehling** [comment]
**In response to line 675:** 
_675: +If either of the two non-Chairman member of the Executive Board are deemed biased, unfit for a position on the Judicial Board, or are involved in the Judicial as an accuser or accused, the Judicial Board will instead be composed of 2 non-Executive Board members randomly chosen from active on-floor members, and the Chairman._

In the event that an e-board member is accused, I do not believe selecting a random member of house to fill their slot is the best course of action. Perhaps instantiating a list of approved arbitrators ("approved" meaning voted on by house as being worthy of the position) would help to guard against this scenario by replacing the defendant with an approved, impartial third party.

**Comment by metal-crow** [comment]
That is an excellent idea, thank you! I didn't like the random selection, but hadn't come up with a better idea.

**Comment by mbillow** [comment]
Regardless of my opinions on this: `:s/Department of/Center for/g`.

**Comment by brandonhudson** [comment]
I am incredibly opposed to this for several reasons:
- Given our continued dialogue with residence life over the past few years, we are **heavily** moving away from using our Judicial process to handle such events, and are instead immediately directing the situation to residence life. This is not our eboard's decision on the matter, but rather guidance coming directly from our reslife advisor and other members of the RIT staff.
  - I am personally of the opinion that once residence life concludes their work and an outcome is reached, we can then look at what they discovered and work hand-in-hand with our advisor to take care of internal actions that might be needed.
- I am **extremely** opposed to codifying any form of immediate membership removal that relies solely on one individual's testimony of discomfort. Removal of membership should be used with extreme care and should **never** be written as a pseudo-automated process in our constitution.
  - The cases you are citing as extreme here should never be acted upon in a "Judicial-first" manner - all of these should be immediately sent to residence life without any of our intervention.

**Comment by mbillow** [comment]
Now for my opinions... Eboard last year spent an incredibly long time talking about this and how we should be using Judicial Boards going forward. We made sure to document that conversation for future generations, but basically the gist of it was: Any issue that involves illegal activity or harassment is not an issue CSH should be involved in resolving. The Center for Residence Life (CRL) in collaboration with the Center for Conduct and Conflict Resolution have a far more appropriate skill set in dealing with these issues. There are educations programs that people can be referred to, counselors that can be made accessible, etc. - none of which CSH has the resources to provide (nor should we). 

Codifying this though, can be hard. The purpose for the constitution is to be a guiding document that we refer to in our decision making processes, not a playbook with all of the answers. Harassment and assault are not transgressions that should be taken lightly and if they are handled incorrectly can mature into even larger issues. This document should be about operational guidelines and with that being said, if you want to codify the inappropriate nature of using a Judicial Board in these cases, we can talk about that further. As for handling these issues internally, I am unequivocally apposed.

**Comment by speakerbug** [comment]
Just my two-cents about this whole amendment. It's possible this is completely irrelevant as I don't know the current state of floor, but thought I'd still post anyway.

I agree with @stevenmirabito that putting something like this in writing is putting too much responsibility for dealing with these kinds of things onto CSH. I do, however, understand the want for behavior like this to not be tolerated and handled appropriately. I'm not really sure of the current issues/temperament of floor, but I'm assuming this was proposed due to some sort of incident.

If this is the case and you feel like it's being incorrectly handled, you should reach out to someone in ResLife. I also think that instead of an amendment, a sort of "cheat sheat" could be created for dealing with issues like these with a partnership between ResLife and CSH. It would be super simple to create a document that has a list of all the people within RIT you can contact about certain issues that arise (like bullying/abuse, but also not just limited to that) and make sure everyone knows those resources exist. Announce it at house meeting and put the document up around floor. This will probably also be more beneficial in the end too because let's be honest, only a handful of people actually read this constitution, but everyone would read a quick "cheat sheet".

These resources are available at RIT because college is hard and bad things happen and some issues are just beyond the scope of EBoard. For me, bullying/abuse lands in that category.

TL;DR: Don't make this an amendment and instead create a "cheat sheet" with all the resources on campus to handle these types of issues and spread it around floor.

**Comment by metal-crow** [comment]
I agree with much of the criticism posted here, and as such will be dropping the amendment as is. I may rewrite it in the future to enforce what seems to be the majority opinion (all cases such as these should be immediately sent to res life), but it will be dramatically rewritten.
